rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/health/10_basic.yml

```diff
@@ -0,0 +1,14 @@
+---
+"cluster health basic test":
```

Comment:
do we want to add any tests for RED cases?

Reply:
It's very difficult to test this at the yaml level. The PR will require some type of Integ test for that master missing indicator though. I will add it before merge.